bot/resources/tags/nomodule.md

```diff
@@ -0,0 +1,15 @@
+**ModuleNotFoundError**
+
+If you've installed a package but you're getting a ModuleNotFoundError, it's likely that the environment where your code is running is different from the one where you did the installation.
```

Comment:
What do you think about briefly clarifying here that the error happens upon importing the purportedly installed package?